CHANGELOG.md

```diff
@@ -12,6 +15,7 @@
 * Add some View Nodes - View.HideCategoriesTemporary, View.HideElementsTemporary, View.IsolateCategoriesTemporary, View.IsolateElementsTemporary.
 * Improve DuplicateSheet Node - add "duplicateWithContents" option, add default suffix value when prefix & suffix are both empty.
 * Improve DuplicateSheet Node - Set Sheet information when duplicating; Improve View Temporary Nodes.
+* Add 4 UI Nodes - View Duplicate Options, View Disciplines, View DisplayStyles, View PartsVisibilitys
```

Comment:
I can't find these 4 UI Nodes in this PR.

Reply:
It was forgotten to be recorded in the previous PR, now it is added.